src/utils/test/template.ts

```diff
@@ -19,7 +19,8 @@ export function getTemplateAfterAspectInvocation(stack: GuStack): Template {
   }
 
   const { artifacts } = app.synth();
-  const cfnStack = artifacts.find((_): _ is CloudFormationStackArtifact => _ instanceof CloudFormationStackArtifact);
```

Comment:
This was causing some of our new unit tests to fail, which added to my confusion for a while!